html-tables

John-bess4 opened the pull request:
<img width="771" alt="Screen Shot 2023-05-16 at 7 30 25 AM" src="https://github.com/John-bess4/c0523-code-solutions/assets/126554940/d142eefe-5fd7-45c2-81b4-1e07e6818172">

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
```
The name of the exercise needs to go here!

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+          <th scope="col">Grade</th>
+        </tr>
+        <tr>
+          <th scope="row">Grace Hopper</th>
```
This should be a `td` element

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+        </tr>
+
+        <tr>
+          <th scope="row">Alan Turing</th>
```
This should be a `td` element

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+        </tr>
+
+        <tr>
+          <th scope="row">Douglas Crockford</th>
```
This should be a `td` element

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+        </tr>
+
+        <tr>
+          <th scope="row">Tim Berners-Lee</th>
```
This should be a `td` element

shawnkost commented on html-tables/index.html:
```diff
@@ -0,0 +1,47 @@
+        </tr>
+
+        <tr>
+          <th scope="row">Linus Torvalds</th>
```
This should be a `td` element

shawnkost reviewed (changes requested):
Some changes requested here John!

John-bess4 commented:
<img width="778" alt="Screen Shot 2023-05-28 at 4 48 55 PM" src="https://github.com/John-bess4/c0523-code-solutions/assets/126554940/c538fc29-0ac4-4e41-898a-42c99f27a837">


Fixed the TH and TD issues and also fixed the title issue as well.
